examples/webgl_loader_pdb.html

```diff
@@ -108,16 +109,16 @@
 
 			function init() {
 
-				camera = new THREE.PerspectiveCamera( 70, window.innerWidth / window.innerHeight, 1, 5000 );
-				camera.position.z = 1500;
-
 				scene = new THREE.Scene();
 
-				var light = new THREE.HemisphereLight( 0xffffff, 0x000000 );
-				light.position.set( 1, 1, 1 );
```

Comment:
Not exactly. I removed the hemisphere light and replaced it with a point light. Remember, the new hemisphere light will have not specular component -- it will be flat. I was trying to keep the demo appearance about the same as it was previously.